webdriver/tests/element_send_keys/form_controls.py

```diff
@@ -42,6 +43,20 @@ def test_input_append(session):
     assert element.property("value") == "abc"
 
 
+def test_input_append_with_focus(session):
```

Comment:
We should also have this test for the content editable elements. See the other file in this folder.

Reply:
I think this still has to be done.